File: test/data/src/py/ShootNumber.py

oldCanopyShootNumber = canopyShootNumber
emergedLeaves = int(max(1, ceil(leafNumber - 1)))
shoots = fibonacci(emergedLeaves)
canopyShootNumber = min(shoots * sowingDensity, targetFertileShoot)
averageShootNumberPerPlant = canopyShootNumber / sowingDensity       
if (canopyShootNumber != oldCanopyShootNumber):
    tilleringProfile.append(canopyShootNumber - oldCanopyShootNumber)         
tillerNumber = len(tilleringProfile)     
for i in range(len(leafTillerNumberArray),int(ceil(leafNumber))):
    leafTillerNumberArray.append(tillerNumber)
    

File: test/data/phenology/crop2ml/algo/f90/ShootNumber.f90

    !use crop2mlModules
        REAL::oldcanopyShootNumber, calc
        INTEGER:: emergedLeaves, i
        oldcanopyShootNumber = canopyShootNumber
        emergedLeaves = INT(MAX(1, CEILING(leafNumber - 1)))
        shoots = fibonacci(emergedLeaves)
        canopyShootNumber = MIN(shoots * sowingDensity, INT(targetFertileShoot))
        averageShootNumberPerPlant = canopyShootNumber / sowingDensity
        IF (canopyShootNumber /= oldcanopyShootNumber) THEN
            CALL AddToList(tilleringProfile,canopyShootNumber - oldcanopyShootNumber)
        END IF
         tillerNumber = SIZE(tilleringProfile)
        DO i=SIZE(leafTillerNumberArray), CEILING(leafNumber)-1
            CALL AddToListInt(leafTillerNumberArray,tillerNumber)
        END DO